src/FormBuilder/resources/js/form-builder/src/components/canvas/FormSettingsContainer/components/Menu.tsx

```diff
@@ -0,0 +1,63 @@
+import {__} from '@wordpress/i18n';
+import {chevronLeft, chevronRight} from '@wordpress/icons';
+import {Icon} from '@wordpress/components';
+import {useFormSettingsContext} from '@givewp/form-builder/components/canvas/FormSettingsContainer';
+import {
+    navigateBackInMenu,
+    updateMenuState,
+} from '@givewp/form-builder/components/canvas/FormSettingsContainer/formSettingsReducer';
+import classnames from 'classnames';
+
+function MenuItem({item}) {
+    const [state, dispatch] = useFormSettingsContext();
+    const isActive = state.activeMenu === item.path;
+    const hasChildren = item.childRoutes && item.childRoutes.length;
+    const isActiveParent = hasChildren && state.activeMenu.split('/').includes(item.path);
+
+    if (item.showWhen && !item.showWhen(item)) {
+        return null;
+    }
+
+    const handleItemClick = () => {
+        dispatch(updateMenuState(hasChildren, item.path));
+    };
+
+    const handleBackClick = () => {
+        dispatch(navigateBackInMenu());
+    };
+
+    return (
+        <li className={classnames({'is-active': isActive, 'has-children': hasChildren})}>
```

Comment:
For consistency, I think we should make these classnames coincide with the exiting hierarchal BEM structure like 
```
givewp-form-settings__menu__item 
givewp-form-settings__menu__item--is-active 
givewp-form-settings__menu__item--has-children
```

Reply:
Resolved: af414a34fe6b3996729d607999b4635ea38b6d35